lisa/nic.py

```diff
@@ -284,6 +310,15 @@ def reload(self) -> None:
         self.nics.clear()
         self._initialize()
 
+    def check_sriov_enabled(self, sriov_enabled: bool) -> None:
+        if sriov_enabled and not self.get_lower_nics():
+            raise LisaException("SRIOV is enabled, but VF is not found.")
+        elif not sriov_enabled and self.get_lower_nics():
+            raise LisaException("SRIOV is disabled, but VF exists.")
+        else:
+            # the enabled flag is consistent with VF presents.
+            ...
+
     @retry(tries=15, delay=3, backoff=1.15)
     def wait_for_sriov_enabled(self) -> None:
```

Comment:
This method is similar to `check_sriov_enabled`. They can be merged on wait or check pci enabled or not.